src/pages/AddItem.jsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from "react";
+import { Flex } from "../styles/styled";
+import "../App.css";
+import {
+  Label,
+  Input,
+  Title,
+  TextArea,
+  Submit,
+  Form,
+  Tag,
+} from "../styles/styled/Additem";
+import ItemFileInput from "../components/ItemFileInput";
+import CancelLogo from "../assets/ic_X@3x.png";
+import styled from "styled-components";
+
+const INITIAL_VALUE = {
+  imageFile: null,
+  title: "",
+  content: "",
+  price: "",
+  tag: "",
+};
+
+const Cancel = styled.img`
+  width: 22px;
+  height: 24px;
+  cursor: pointer;
+`;
+
+function AddItem() {
+  const [value, setValue] = useState(INITIAL_VALUE);
+  const [tags, setTags] = useState([]);
+  const { imageFile, title, content, price, tag } = value;
+
+  const isValid =
+    title !== "" && content !== "" && price !== "" && tags.length > 0;
+
+  const onChange = (name, value) => {
```

Comment:
```suggestion
  const onChange = (name, value) => {
```
어떤걸 onChange 하는건지 이름에 적혀있으면 더 좋을것같네요!